usr/src/uts/common/fs/zfs/spa_history.c

```diff
@@ -255,6 +266,7 @@ spa_history_log_sync(void *arg, dmu_tx_t *tx)
 			    fnvlist_lookup_string(nvl, ZPOOL_HIST_INT_NAME),
 			    fnvlist_lookup_string(nvl, ZPOOL_HIST_INT_STR));
 		}
+		spa_history_log_notify(spa, nvl);
```

Comment:
Why do we only post a sysevent for the "internal" history messages?  I can imagine several reasons, could you add a comment to the code explaining?